### DISCUSSION TITLE
Welcome to SparqlExtension Discussions!

### DISCUSSION THREAD

**Discussion by WaltonG** [OpenRefine/SparqlExtension#10] [Announcements]
## 👋 Welcome!
 We’re using Discussions as a place to connect with other members of our community. We hope that you:
 
 * Ask questions you’re wondering about.
 * Share ideas.
 * Engage with other community members.
 * Welcome others and are open-minded. Remember that this is a community we
   build together 💪.
 
 To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WaltonG** [comment]
Hello everyone, am Walton Goga an outreachy intern working on Implementing a SPARQL Importer for Openrefine. Looking forward to your discussions of ideas for successful realization of the project.

**Comment by ostephens** [comment]
Hi all, I'm Owen, a long time user and some time contributor to OpenRefine. In the past I have worked on projects where a SPARQL importer would have been useful - specifically integration with Wikidata Query. However I don't have any projects like that right now.

Look forward to seeing how this extension develops!
